README.md

```diff
@@ -146,3 +138,17 @@ Learn how in our dedicated documentation pages:
 
 - Migrating from [ActiveUI 4 to 5](https://activeviam.com/activeui/documentation/5.1.0/docs/migrate-from-activeui-4-to-5).
 - Migrating from [ActiveUI 5.0 to 5.1](https://activeviam.com/activeui/documentation/5.1.0/docs/migrate-from-activeui-5.0-to-5.1).
+
+## Adding your own migration code
```

Comment:
I feel like we need to verbalize this in a way that's closer to the end goal of the reader. One does not usually just need to "add code". They have a more final goal in mind.

```suggestion
## Customizing the migration

This paragraph will help you if you maintain an ActiveUI extension, decided to change the state it relies on and are looking to migrate some saved content so that it works with the new version of your extension.
```

Happy to hear a different phrasing.